glean-core/ios/Glean/Metrics/DatetimeMetric.swift

```diff
@@ -0,0 +1,144 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+
+/// This implements the developer facing API for recording datetime metrics.
+///
+/// Instances of this class type are automatically generated by the parsers at build time,
+/// allowing developers to record values that were previously registered in the metrics.yaml file.
+///
+/// The datetime API only exposes the `DatetimeMetricType.set` method, which takes care of validating the input
+/// data and making sure that limits are enforced.
+public class DatetimeMetricType {
```

Comment:
Can you please update the docs [here](https://mozilla.github.io/glean/book/user/metrics/datetime.html) to add the Swift usage?